selfdrive/controls/controlsd.py

```diff
@@ -642,6 +649,8 @@ def state_control(self, CS):
         # TODO use desired vs actual curvature
         if self.CP.steerControlType == car.CarParams.SteerControlType.angle:
           steering_value = actuators.steeringAngleDeg
+        elif self.CP.steerControlType == car.CarParams.SteerControlType.curvature:
+          steering_value = actuators.curvature
```

Comment:
User reported that "steering limit reached" alert never appears - this line is suspect but I tried it both `-ve` and `+ve` and it didn't appear. maybe the issue is somewhere else.